py/server/deephaven/experimental/iceberg.py

```diff
@@ -30,6 +33,39 @@
 _JSnapshot = jpy.get_type("org.apache.iceberg.Snapshot")
 
 
+class IcebergUpdateMode(JObjectWrapper):
+    """
+    This class specifies the update mode for the Iceberg table to be loaded into Deephaven. The modes are:
+
+    - `STATIC`: The table is loaded once and does not change
+    - `MANUAL_REFRESHING`: The table can be manually refreshed by the user.
+    - `AUTO_REFRESHING`: The table will be automatically refreshed at a specified interval (use
+            `auto_refreshing(auto_refresh_ms: int)` to specify an interval rather than use the system default
+            of 60 seconds).
+    """
+    j_object_type = _JIcebergUpdateMode
+
+    def __init__(self, mode: _JIcebergUpdateMode):
+        self._j_object = mode
+
+    @classmethod
+    def auto_refreshing(cls, auto_refresh_ms: int) -> IcebergUpdateMode:
+        """
+        Creates an IcebergUpdateMode with auto-refreshing mode enabled using the provided refresh interval.
+
+        :param auto_refresh_ms (int): the refresh interval in milliseconds.
+        """
+        return IcebergUpdateMode(_JIcebergUpdateMode.autoRefreshing(auto_refresh_ms))
+
+    @property
+    def j_object(self) -> jpy.JType:
+        return self._j_object
+
+IcebergUpdateMode.STATIC=IcebergUpdateMode(_JIcebergUpdateMode.STATIC)
+IcebergUpdateMode.MANUAL_REFRESHING=IcebergUpdateMode(_JIcebergUpdateMode.MANUAL_REFRESHING)
+IcebergUpdateMode.AUTO_REFRESHING=IcebergUpdateMode(_JIcebergUpdateMode.AUTO_REFRESHING)
```

Comment:
I don't think this style matches with the rest of the pythoAPI.  I think enums have been used for cases like this.  @jmao-denver should comment.

e.g. https://github.com/deephaven/deephaven-core/blob/3bd5587310b73c9e81bb8b53a55c96aa2a34db06/py/client/pydeephaven/updateby.py